test/unit/validation.spec.js

```diff
@@ -0,0 +1,42 @@
+import {
+  describe,
+  it,
+  expect,
+  moment,
+  isDateFuture,
+  isDateValid,
+  isDescriptionAdequate,
+  isLocationValid
+} from '../helpers';
+
+describe('Validations:', () => {
+  describe('Date:', () => {
+    describe('isDateValid:', () => {
+      it('should check the validity of a date', () => {
+        expect(isDateValid('25-01-17')).true;
+        expect(isDateValid('25-44-1998')).false;
+      });
+    });
```

Comment:
These are just dummy tests, one is enough.

Reply:
@kimobrian the validation dummy specs I wrote used real tests, 16 is the happy path while 17 tests for the not-so-happy path. intentionally done to be used in prepping the doc you asked for.